src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -756,21 +819,23 @@ private static int compareImports(String import1, String import2) {
     }
 
     /**
-     * Counts empty lines before given.
-     * @param lineNo
-     *        Line number of current import.
-     * @return count of empty lines before given.
+     * Counts empty lines between given parameters.
+     * @param fromLineNo
+     *        One-based line number of previous import.
+     * @param toLineNo
+     *        One-based line number of current import.
+     * @return count of empty lines between given parameters, inclusive,
+     *        eg., [fromLineNo, toLineNo].
      */
-    private int getCountOfEmptyLinesBefore(int lineNo) {
+    private int getCountOfEmptyLinesBetween(int fromLineNo, int toLineNo) {
```

Comment:
The original logic counts only empty lines before given import statement. 
Current counts between import statements.